docker-compose.dev.yml

```diff
@@ -93,6 +93,9 @@ services:
       BASE_URL: "http://localhost/streams/ldes/"
     ports:
       - "6666:80"
```

Comment:
want to be able to run ldes-delta-pusher as a debug compose on the side if we want to